wavefront/release/VERSION

```diff
@@ -1,2 +1,2 @@
 APP_VERSION="1.7.5"
-CHART_VERSION="1.7.10"
+CHART_VERSION="1.7.11"
```

Comment:
I don't think we want to make this change The open source version is currently version: 1.7.10 and the openshift version is 1.7.11